### PR TITLE
fix: add learn web dev survey banner

### LINKED ISSUE #2858: Aspiring web devs banner
Copy provided by @chrisdavidmills 

I propose that the new banner for the education survey could read something like this:

Learning web development survey
Help us understand how to make MDN better for beginners (5 minute survey)
Take the survey

### PR DESCRIPTION
**Pull request by schalkneethling**
Adds the new learning web development survey. Only shown on desktop and tablet.

Screenshot 2021-02-09 at 16 56 21

fix #2858

### CODE REVIEW

**Comment by schalkneethling** [comment]
The test failure looks to be related to https://github.com/mdn/yari/pull/2857

File: client/src/banners/active-banner.tsx [hunk 1]
```diff
@@ -82,45 +87,28 @@ function Banner(props: BannerProps) {
...
-      title={"MDN Survey"}
+      id={COMMON_SURVEY_ID}
+      title={"Learning web development survey"}
       copy={
-        "Help us understand the top 10 needs of Web developers and designers."
+        "Help us understand how to make MDN better for beginners (5 minute survey)"
       }
       cta={"Take the survey"}
-      url={"https://qsurvey.mozilla.com/s3/Developer-Needs-Assessment-2019"}
+      url={"https://www.surveygizmo.com/s3/6175365/59cfad9c04cf"}
```
Comment (peterbe) [inline client/src/banners/active-banner.tsx:101]: Nit: there's no need for curly brackets with it's just a string. For example...
```suggestion
      url="https://www.surveygizmo.com/s3/6175365/59cfad9c04cf"
```

File: client/src/ga-context.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { DEBUG_GOOGLE_ANALYTICS } from "./constants";
 
 export type GAFunction = (...any) => void;
 
-export const CATEGORY_MONTHLY_PAYMENTS = "monthly payments";
+export const CATEGORY_LEARN = "learning web development";
```
Comment (peterbe) [inline client/src/ga-context.tsx:8]:
```suggestion
export const CATEGORY_LEARNING_SURVEY = "learning web development";
```
maybe?

**Review by peterbe — APPROVED** [review]
Tested it (using waffle Switches (instead of waffle Flags)). And it works.

**Comment by peterbe** [comment]
@schalkneethling Can you review and merge https://github.com/mdn/yari/pull/2857 and then merge in `master` into this branch to check that the tests pass. Then merge.

**Comment by schalkneethling** [comment]
> @schalkneethling Can you review and merge #2857 and then merge in `master` into this branch to check that the tests pass. Then merge.

I saw Ryan merged that one. So, let me sort out this piece. Thank you, Peter.

**Pull request merged**